### PR TITLE
Better management of plural in relations

### PR DESCRIPTION
**Pull request by cicoub13**
Hello,

I've changed all declarations and uses of `<% someEntityName %>s` by `<% someEntityNamePlural %>`. Plural version of entity or relation name is calculated before in js with **pluralize**.

This PR is a completion of https://github.com/jhipster/generator-jhipster/pull/2719

Please feel free to comment and criticize.

Thanks,

### CODE REVIEW

**Comment by cicoub13** [comment]
I didn't add question during creation of the Entity to ask custom plural name.

**Comment by cicoub13** [comment]
In relation with https://github.com/jhipster/generator-jhipster/issues/1895 and https://github.com/jhipster/generator-jhipster/issues/1029 too

File: generators/entity/templates/src/main/java/package/web/rest/dto/_EntityDTO.java [hunk 1]
```diff
@@ -118,20 +119,23 @@ public void setId(<% if (databaseType == 'sql') { %>Long<% } %><% if (databaseTy
...
         otherEntityNameCapitalized = relationships[idx].otherEntityNameCapitalized,
         otherEntityFieldCapitalized = relationships[idx].otherEntityFieldCapitalized,
         relationshipNameCapitalized = relationships[idx].relationshipNameCapitalized,
+        relationshipNameCapitalizedPlural = relationships[idx].relationshipNameCapitalizedPlural,
         ownerSide = relationships[idx].ownerSide;
         if (relationshipType == 'many-to-many' && ownerSide == true) { _%>
 
-    public Set<<%= otherEntityNameCapitalized %>DTO> get<%= relationshipNameCapitalized %>s() {
-        return <%= relationshipFieldName %>s;
+    public Set<<%= otherEntityNameCapitalized %>DTO> get<%= relationshipNameCapitalized %>() {
```
Comment (deepu105) [inline generators/entity/templates/src/main/java/package/web/rest/dto/_EntityDTO.java]: This should be plural as well

Comment (cicoub13) [inline generators/entity/templates/src/main/java/package/web/rest/dto/_EntityDTO.java]: Corrected, thanks

**Pull request merged**